apps/web/src/pages/liquidity/[tokenId].tsx

```diff
@@ -990,7 +989,7 @@ function PositionHistoryRow({
   currency0,
   currency1,
 }: {
-  chainId: ChainId
+  chainId: ChainId | undefined
```

Comment:
just a suggestion, if it's possibly undefined, we can use the optional mark
```suggestion
  chainId?: ChainId
```